server/webapp/WEB-INF/rails.new/app/views/admin/tasks/fetch/_external_artifact_config.erb

```diff
@@ -0,0 +1,19 @@
+<div class="form_item_block" id="artifact_id_input">
+  <%= label_tag random_dom_id('pipeline_'), required_label_text('Artifact ID') -%>
+  <%= scope[:var_form].text_field com.thoughtworks.go.config.FetchPluggableArtifactTask::ARTIFACT_ID, {:name => "task[artifactId]", :id => "task_artifactId"} %>
+  <div class="contextual_help has_go_tip_right" title="The ID of the artifact."></div>
+  <%= error_message_on(@task, com.thoughtworks.go.config.FetchPluggableArtifactTask::ARTIFACT_ID, :css_class => "form_error") %>
+</div>
+
+<div class="form_item_block artifact_plugin_selection hidden">
+  <span>Could not determine the plugin from the store. Please choose a plugin to configure the plugin properties.</span>
+  <label>Plugin ID</label>
+  <%= select("FetchArtifact", "pluginId", scope[:artifact_plugin_to_fetch_view].collect {|p| [p.to_hash["name"], p.to_hash["id"]]}, {include_blank: true}) %>
+</div>
+<input type="hidden" name="task[pluginId]" id="task_pluginId"/>
```

Comment:
Shouldn't this make use of `scope[:var_form]` ? That will also generate the same input box with the same name and id.